src/main/java/org/synyx/urlaubsverwaltung/application/ApplicationSettings.java

```diff
@@ -24,6 +24,17 @@
      */
     private Integer daysBeforeRemindForWaitingApplications = 2;
 
+    /**
+     * Activates a notification {daysBeforeRemindForUpcomingApplications} days before upcoming applications
+     */
+    private boolean remindForUpcomingApplications = false;
+
+    /**
+     * Specifies when a reminder for upcoming application should be send to boss / department head /
+     * secondary stage authority
```

Comment:
Why send to boss / department head / secondary stage authority? Its for the applicant, isn't it?